hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/OwnerTransformationsSpec.scala

```diff
@@ -59,7 +59,7 @@ class OwnerTransformationsSpec extends AnyFlatSpec with Matchers {
     output.ssHouseholdDogCount shouldBe Some(2)
     // residence fields
     output.ocPrimaryResidenceState shouldBe Some("OH")
-    output.ocPrimaryResidenceCensusDivision shouldBe Some("Division 3: East North Central")
+    output.ocPrimaryResidenceCensusDivision shouldBe Some(3)
```

Comment:
Should this be 33? If we're extracting 3, we're losing the 2nd digit.

It looks like the other test is checking the double-digit case, so I'd actually say we should revert the source data in this one to be 3.

Reply:
Seconded; looks like it should be 33